common/data_refinery_common/models/jobs.py

```diff
@@ -127,7 +127,7 @@ class Meta:
     failure_reason = models.TextField(null=True)
 
     # If the job is retried, this is the id of the new job
-    retried_job = models.ForeignKey('self', on_delete=models.PROTECT, null=True)
+    retried_job = models.ForeignKey('self', on_delete=models.SET_NULL, null=True)
```

Comment:
If there are new fields on the models, I'd also expect new fields on the serializers